### PR TITLE
Increase clap version to v3

### PR DESCRIPTION
**Pull request by galenguyer**
This way we can use clap::command!() and get information about the version from the Cargo.toml file.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 serde = "1.0.130"
 ncurses = "5.101.0"
 http = "0.2.5"
+clap = { version = "3.1.6", features = ["cargo"] }
 
 [dependencies.isahc]
 version = "1.6.0"
 features = ["json", "spnego"]
+
+[features]
```
Comment (Mstrodl) [inline Cargo.toml]: unintended?

File: Cargo.toml [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-clap = "2.33.3"
 url = "2.2.2"
 serde_json = "1.0.71"
 serde = "1.0.130"
 ncurses = "5.101.0"
 http = "0.2.5"
+clap = { version = "3.1.6", features = ["cargo"] }
```
Comment (Mstrodl) [inline Cargo.toml:15]: we should probably decide if we want this syntax or the syntax below it...

File: Cargo.toml [hunk 3]
```diff
@@ -1,18 +1,21 @@
 [package]
 name = "clink"
+authors = ["Mary Strodl <mstrodl@csh.rit.edu>"]
```
Comment (Mstrodl) [inline Cargo.toml]: Willard!

**Review by Mstrodl — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (galenguyer) [reply Cargo.toml]: Added by https://lib.rs/crates/cargo-edit, it can be removed without issue

File: Cargo.toml (hunk 2 above)
Comment (galenguyer) [reply Cargo.toml:15]: This syntax is what I see a lot more of, especially in the docs

File: Cargo.toml (hunk 3 above)
Comment (galenguyer) [reply Cargo.toml]: Willard wasn't included in the clap authors bit, which is what I copied. I'll add him.

**Review by WillNilges — APPROVED** [review]
:ok: WOMM. 

Please address Mary's issues.

File: Cargo.toml (hunk 2 above)
Comment (Mstrodl) [reply Cargo.toml:15]: I was hoping we could use this syntax

File: Cargo.toml (hunk 1 above)
Comment (Mstrodl) [reply Cargo.toml]: Remove it?

File: src/api.rs [hunk 4]
```diff
@@ -34,7 +34,7 @@ impl API {
     let token = self.get_token()?;
 
     let client = HttpClient::new()?;
-    let request = Request::post("https://drink.csh.rit.edu/drinks/drop")
+    let request = Request::post("http://localhost:8000/drinks/drop")
```
Comment (Mstrodl) [inline src/api.rs]: Unintended?

**Review by Mstrodl — CHANGES_REQUESTED** [review]

File: src/api.rs (hunk 4 above)
Comment (galenguyer) [reply src/api.rs]: oops

**Review by Mstrodl — APPROVED** [review]
:shipit:

**Pull request merged**